nautobot/docs/installation/ubuntu.md

```diff
@@ -4,8 +4,85 @@ This installation guide assumes that you are running Ubuntu version 20.04 on you
 
 ## Install System Packages
 
-Install the prerequisite system libraries and utilities:
+Install the prerequisite system libraries and utilities.
+
+This will install:
+
+- Python 3
+- Pip
+- PostgreSQL database server and client
+- Redis server and client
```

Comment:
Do we need to include `git` as well here like we did on CentOS?

Reply:
Yes.

Reply:
Well. For consistency, yes. But if we can align around a singular deployment pattern that doesn't require Git, all the better.

Reply:
Git is for the Git integration feature, actually :) It's not needed for the installation per se.

Reply:
Right yeah, I guess it wouldn't hurt to pre-install it in that case.